Gemfile

```diff
@@ -39,10 +41,12 @@ group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
   gem 'rubocop-rails'
   gem 'travis'
+  gem 'factory_bot_rails'
+  gem 'faker'
```

Comment:
I duplicated this in a couple places based on googling why Heroku couldn't find the Faker gem to seed the database. Still haven't quite figured it out, but maybe once this branch is actually merged it will recognize the gems.

Reply:
Agreed, cant hurt could help, good thinking!